### ISSUE TITLE
Add previews

### ISSUE THREAD

**Issue by FinnEatsCandy** [KirovAir/TwilightBoxart#15]
Great application but it would be useful to see examples of the boxart size menu
![image](https://user-images.githubusercontent.com/69067900/209484416-900912b2-4d50-4c6e-be43-ca08c2d7958c.png)

**Comment by KirovAir** [comment]
I'm currently rewriting the whole app and will probably add something like this.

**Comment by Agusum** [comment]
@KirovAir Great to see that you're working again on the app! Is there any beta version I can test? I'm using macOS. It'd be awesome to have an app with UI for Mac or even better, to use it from the 3DS itself! I've tried with the [TWiLightMenu-Updater Modified version](https://github.com/KirovAir/TWiLightMenu-Updater/releases) but it's not the most stable app.

It would be good to include what each of the different borders look like in the readme.

**Comment by KirovAir** [comment]
Good idea! 2.0 just shipped with a fresh UI (https://boxart.kirovair.com), but it doesn't show size/border previews yet either. Leaving this open, I'll check it out for a next release.
